memebot/memebot.py

```diff
@@ -173,17 +174,26 @@ async def skip(command_arg: str, channel: discord.TextChannel, player_config: Pl
         )
     return skip
 
+async def queue_display(command_arg: str, channel: discord.TextChannel, player_config: PlayerConfig):
+    if len(player_config.player.playback_queue) == 0:
+        return await channel.send('Nothing in queue. Add a song with !play', delete_after=30)
+    queue = '\n'.join(f'[{idx}] {item.title}' for idx,item in enumerate(player_config.player.playback_queue))
+    return await channel.send(f'Up next:\n{queue}', delete_after=60)
```

Comment:
would be nice if we only kept one queue message up at a time to avoid any confusion since the indexes will change as the player goes through songs. though that's probably overkill for now

Reply:
Feel like
1. If you can't navigate a queue, you're dumb
1. If you can't figure out to run `!queue` again just to be sure, you're dumb
1. and finding the previous queue message involves annoying things